Riot/Modules/Settings/IdentityServer/SettingsIdentityServerViewController.swift

```diff
@@ -142,7 +142,6 @@ final class SettingsIdentityServerViewController: UIViewController {
         self.identityServerLabel.text = VectorL10n.identityServerSettingsTitle
 
         self.identityServerTextField.placeholder = VectorL10n.identityServerSettingsPlaceHolder
-        self.identityServerTextField.keyboardType = .URL
```

Comment:
It's not a problem here to keep `self.identityServerTextField.keyboardType = .URL` even if it's set in the xib

Reply:
Ah ok. Just seemed a bit tidier to keep all the settings in the same place. I can put it back if you prefer?